utils/publish-canary.js

```diff
@@ -83,6 +83,7 @@ exec('git diff --name-only HEAD HEAD^')
       `--canary`,
       `--preid ${preid}`,
       `--dist-tag ${distTag}`,
+      `--max-buffer=999999999`,
```

Comment:
Was this issue fixed with node 12?

Reply:
Not sure, I'm not familiar with why this is here

Reply:
It reminds me of this: https://github.com/Workday/canvas-kit/pull/908/files#diff-53d182f2fc69a23673a933acbee9ac295c91e80b1ccc93e316b87b8f92a55067L90-L95

We were able to remove the max buffer override by changing the version of node we're using in TravisCI from 10 to 12

Reply:
Hmm, I think this may have came in from a rebase/merge so I will remove it.